test/vm/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+set(UBUNTU_CONTAINER1 "vm-ubuntu2004:latest")
+set(UBUNTU_CONTAINER2 "vm-ubuntu2204:latest")
+set(UBUNTU_CONTAINER3 "vm-ubuntu2310:latest")
+set(DEBIANBUSTER_CONTAINER "vm-debianbuster:latest")
+set(VM_PROVIDER_VBOX "virtualbox")
+set(VM_NAMES_VBOX_VAGRANT_CENTOS7 "centos7")
+set(VM_NAMES_VBOX_VAGRANT_UBUNTU "ubuntu")
+set(VM_NAMES_VBOX_VAGRANT_AMAZONLINUX2 "amazonlinux2")
+set(VM_NAMES_VBOX_VAGRANT "centos7 ubuntu amazonlinux2")
+
+set(VM_CONTAINERS
+	${UBUNTU_CONTAINER1} 
+	${UBUNTU_CONTAINER2} 
+	${UBUNTU_CONTAINER3} 
+	${DEBIANBUSTER_CONTAINER} 
+	falcosecurity/falco-builder:latest
+)
+
+add_custom_target(vm-container
+	COMMAND time docker build -f ${CMAKE_CURRENT_SOURCE_DIR}/containers/ubuntu2004.Dockerfile -t ${UBUNTU_CONTAINER1} .;
```

Comment:
Just a note: when `time` is not installed as binary, the shell `sh` `time` builtin seems not to be picked, ending up to:
```
time: no such file or directory
```
Experienced and fixed On Arch Linux, by installing separately time package.